### ISSUE TITLE
Confusing parameter name $classes

### ISSUE THREAD

**Issue by mvorisek** [doctrine/orm#8358]
See `SchemaTool::getUpdateSchemaSql(array $classes)` method.

`$classes` represents array of `ClassMetadata`, no string class names.

**Comment by greg0ire** [comment]
Indeed: https://github.com/doctrine/orm/blob/404edd418b983d14a35c62ab297a4ead66ed2063/lib/Doctrine/ORM/Tools/SchemaTool.php#L882-L905

https://github.com/doctrine/orm/blob/404edd418b983d14a35c62ab297a4ead66ed2063/lib/Doctrine/ORM/Tools/SchemaTool.php#L137-L161

The phpdoc could also be improved by using `@psalm-param ClassMetadata[] $classes`. Please send a PR.

**Comment by mvorisek** [comment]
I also advice to rename the params to `$metadatas` as used in other method prototypes.
